### DISCUSSION TITLE
How to add iamges in config.js file

### DISCUSSION THREAD

**Discussion by mohammadRezaeian** [dinhanhthi/dinhanhthi.com#37] [Q&A] [answered]
hi i wanted add new images for selection in skill part 
but when i trying for it i show it you use file config.js but i don't know what is it ?
and how can i must config it for new images ?
and is it important i write images in config.js ?

thank you.

**Comment by dinhanhthi** [comment]
There is no config file for skill part, I don't understand what you are talking about. If you want to add a new skill, add the image in `notes/img/about/` and add the information to `notes/_data/skills.json`. Just see what I've done and copy the same ideas.

Please note that, `notes/` is a separated repository which must be cloned before running the project.

**Reply by mohammadRezaeian** [reply]
i mean this file `/src/fontello/config.json`
it's points out to images in these files `/src/fontello/svg/`

**Reply by dinhanhthi** [reply, marked as answer]
If you want to use fontello, you can use this. Otherwise, just forget it. It doesn't take any roll in the skills section.

Again, in the README of the project, I write about this: https://github.com/dinhanhthi/dinhanhthi.com#update-fontello-icons

**Reply by mohammadRezaeian** [reply]
thank you and i am so sorry i forget 
it exists in readme.
